fixity/reporting.py

```diff
@@ -51,7 +61,13 @@ def post_pre_scan_report(aip, start_time, report_url, report_auth=(), session_id
     return True
 
 
-def post_success_report(aip, report, report_url, report_auth=(), session_id=None):
+def post_success_report(
+    aip: str,
+    report: Optional[Report],
+    report_url: Optional[str],
```

Comment:
Why are these `Optional`?

Reply:
Some tests are using report=None.

Reply:
What do you think if we focus on the actual use from the `fixity` package and adjust the tests accordingly if needed?

Reply:
ya let me adjust the tests which focus on the actual use of the codes.